src/main/java/org/opentripplanner/updater/vehicle_rental/datasources/GbfsGeofencingZoneMapper.java

```diff
@@ -52,6 +52,9 @@ private GeofencingZone toInternalModel(GBFSFeature f) {
       return null;
     }
     var name = Objects.requireNonNullElseGet(f.getProperties().getName(), () -> fallbackId(g));
+    if (name.isEmpty()) {
```

Comment:
```suggestion
    if (StringUtils.hasValue(name)) {
```

If you think that you made things idiot proof, the universe will come up with a better idiot. :)